### PR TITLE
[Merged by Bors] - Moved benchmarks to a single place

### LINKED ISSUE #1472: Move all benchmarks to the root level `benches` folder
Location: https://github.com/bevyengine/bevy/tree/master/benches

There are currently several benchmarks scattered elsewhere, and they should be consolidated. 

During this process, we need to be sure we remove the relevant benchmark dev-dependencies from the "real" crates, in order to reduce their dependency trees and improve compile and CI times.

[Discord](https://discord.com/channels/691052431525675048/692572690833473578/812059596054200340) context.

### PR DESCRIPTION
**Pull request by YohDeadfall**
Closes #1472.

### CODE REVIEW

**Comment by cart** [comment]
make sure you remove the dev dependencies from https://github.com/bevyengine/bevy/blob/master/crates/bevy_ecs/Cargo.toml

**Comment by YohDeadfall** [comment]
Already did, but not pushed yet. There's another thing left, `cargo build` succeeds for benchmarks while `cargo bench` fails with compilation errors. Fixing it.

File: benches/benches/bevy_ecs/stages.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-use bevy_ecs::{prelude::*, Stage};
```
Comment (Jerbrone) [inline benches/benches/bevy_ecs/stages.rs:1]: ဖဟ

Comment (Jerbrone) [reply benches/benches/bevy_ecs/stages.rs:1]: help

**Review by Jerbrone — COMMENTED** [review]
idiot

**Pull request closed without merging**